### PR TITLE
[UXIT-1554] GovHub Page

### PR DESCRIPTION
**Pull request by mirhamasala**
## 📝 Description

This pull request introduces a new "GovHub" page to the governance section of the website, including updates to constants, metadata, and testing files. It also adds a new `GovHub` section to the `QuickStart` section of `/governance`.

- **Type:** New feature

### New GovHub Page

* **`src/app/governance/govhub/page.tsx`**: Added a new page component for "GovHub" with sections for "PowerVote" and "Fil Poll" tools, including metadata and structured data scripts.

### Constants and Metadata Updates

* **`src/app/_constants/paths.ts`**: Added a new path for "GovHub" to the `PATHS` constant.
* **`src/app/_constants/siteMetadata.ts`**: Updated governance URLs to include "Fil Poll" and "PowerVote".

### Content and Configuration

* **`src/content/pages/govhub.md`**: Created a new markdown file for "GovHub" content, including header and SEO metadata.
* **`public/admin/config.yml`**: Added a new collection entry for "GovHub" in the CMS configuration.

### Testing

* **`cypress/e2e/critical/govhub_page_spec.cy.ts`**: Added Cypress tests to verify the metadata and links on the "GovHub" page.

## 📌 To-Do Before Merging

- [ ] Add images
- [ ] Style Tools section

## 🧪 How to Test

- **Setup:**
- **Steps to Test:**
  1. Go to `/governance/govhub`
  2. Go to `/governance` - Scroll down to `Quickstart`

### `/governance/govhub`

CleanShot 2024-10-07 at 08 38 20@2x

---

### `/governance`

CleanShot 2024-10-07 at 08 43 38@2x

## 🔖 Resources

- [Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=213-2524&t=GaVnd3weUeSBEZEv-4)

### CODE REVIEW

**Comment by barbaraperic** [comment]
We are not adding the CTA buttons?
Screenshot 2024-10-07 at 12 27 37

**Comment by barbaraperic** [comment]
Just to double check - this is where the user lands on `Explore PowerVote` link?

Screenshot 2024-10-07 at 12 25 48

File: src/app/governance/govhub/data/ToolsData.tsx
Comment (barbaraperic) [inline src/app/governance/govhub/data/ToolsData.tsx:1]: We usually have data files in camelCase? toolsData.tsx

File: src/app/_components/SplitContentCard.tsx
Comment (barbaraperic) [inline src/app/_components/SplitContentCard.tsx:1]: Love what you did here! I find the naming a bit vague tho - how about something with the layout? `AlternatingLayoutCard`

Comment (CharlyMartin) [reply src/app/_components/SplitContentCard.tsx:1]: I find `AlternatingLayoutCard` a bit clearer than `SplitContentCard` indeed.

File: src/app/governance/govhub/components/ToolsCardContent.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div className="flex flex-col gap-6">
+      <div>
+        <Heading tag="h3" variant="lg">
+          {title}
+        </Heading>
+        <p className="mb-6 mt-3">{description}</p>
+        <Heading tag="h4" variant="md">
+          Purpose
+        </Heading>
```
Comment (CharlyMartin) [inline src/app/governance/govhub/components/ToolsCardContent.tsx:30]: The spacing between purpose and the texts below seems smaller on Figma: https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=6695-92069&t=gwKMxyp7Br40sRQl-4

They also seem to use slightly different font weight? Not sure.

File: src/app/governance/govhub/components/ToolsCardContent.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <ul className={clsx(listItems.length > 1 && 'list-disc pl-6', 'mt-2')}>
```
Comment (CharlyMartin) [inline src/app/governance/govhub/components/ToolsCardContent.tsx:31]: I like to put the permanent classes first, then the optional ones :)

```suggestion
        <ul className={clsx('mt-2', listItems.length > 1 && 'list-disc pl-6')}>
```

File: src/app/_components/SplitContentCard.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  image,
+  as: Tag = 'div',
+  children,
+}: SplitContentCardProps) {
+  const isImagePositionOdd = index % 2 === 1
+
+  return (
+    <Tag
+      className={clsx(
+        'flex flex-col gap-4 lg:flex-row lg:gap-6',
```
Comment (CharlyMartin) [inline src/app/_components/SplitContentCard.tsx:29]: In Figma, the gap between the image and the content (`BasicCard`) is always the same as the grid gap.

Here, the gap is set to `gap-4 lg:gap-6` while the grid gap, defined in `PageSection`, is `gap-6`. Hence, on screens smaller than `lg`, this results in wider gaps between cards than within the cards.

Does that make sense?

Not sure if it's a fix for the PR, just flagging it :)

**Review by CharlyMartin — CHANGES_REQUESTED** [review]
Aside from a few minor comments, this is looking really good, Mirha!

File: src/app/governance/govhub/components/ToolsCardContent.tsx (hunk 2 above)
Comment (mirhamasala) [reply src/app/governance/govhub/components/ToolsCardContent.tsx:31]: Makes sense.

File: src/app/governance/govhub/components/ToolsCardContent.tsx (hunk 1 above)
Comment (mirhamasala) [reply src/app/governance/govhub/components/ToolsCardContent.tsx:30]: Yes, it's true. I have an [open request pending on Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA?node-id=6621-139401#975778838). Let's go with what we have for now.

File: src/app/_components/SplitContentCard.tsx
Comment (mirhamasala) [reply src/app/_components/SplitContentCard.tsx:1]: I went with `AlternatingImageCard` 🙏🏼

**Comment by mirhamasala** [comment]
> We are not adding the CTA buttons? @barbaraperic 

I don't think they were there before. 😂

File: src/app/_components/SplitContentCard.tsx (hunk 3 above)
Comment (mirhamasala) [reply src/app/_components/SplitContentCard.tsx:29]: This seems like the intended behavior to me—to have more space between two `AlternatingImageCards` on smaller screens than between the elements within a single `AlternatingImageCard`. I believe adding even more space could be beneficial to make it clearer which image belongs to which card. I’m happy to align this with the design.

**Pull request merged**